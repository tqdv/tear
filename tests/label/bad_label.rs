use tear::twist;

fn bad_label_syntax() {
	twist! { -label 'a 'b | 1 }
}

fn main () {}