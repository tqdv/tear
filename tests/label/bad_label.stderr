error: Bad label syntax: 'a 'b,
 --> $DIR/bad_label.rs:4:2
  |
4 |     twist! { -label 'a 'b | 1 }
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
